fieldservice/models/res_partner.py

```diff
@@ -7,8 +7,16 @@
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
+    type = fields.Selection(selection_add=[("fsm_location", "Location")])
```

Comment:
14.0 Requires ondelete= attribute. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0